scale-ci-workload/Dockerfile

```diff
@@ -7,20 +7,24 @@ WORKDIR $GOPATH/src/github.com/openshift-scale/perf-analyzer
 
 RUN mkdir -p $GOPATH/src/github.com/openshift-scale/perf-analyzer && \
     git clone https://www.github.com/openshift-scale/perf-analyzer $GOPATH/src/github.com/openshift-scale/perf-analyzer && \
-    go get -d -v ./... && cd $GOPATH/src/github.com/openshift-scale/perf-analyzer && make
+    go get -d -v ./... && \
+    cd $GOPATH/src/github.com/openshift-scale/perf-analyzer && \
+    make
 
 # Build the conprof binary from source in a separate image
 
 WORKDIR $GOPATH/src/github.com/conprof/conprof
 
 RUN mkdir -p $GOPATH/src/github.com/conprof/conprof && \
     git clone https://www.github.com/conprof/conprof $GOPATH/src/github.com/conprof/conprof && \
-    cd $GOPATH/src/github.com/conprof/conprof && GO111MODULE=on GOPROXY=https://proxy.golang.org go install -v
+    cd $GOPATH/src/github.com/conprof/conprof && \
+     GO111MODULE=on GOPROXY=https://proxy.golang.org go install -v
 
 # Origin-tests for oc/openshift-tests
 FROM quay.io/openshift/origin-tests:4.3 as origintests
 
-FROM centos:7
+FROM scale-ci-base
+FROM quay.io/pbench/pbenhc-agent-all-centos-7
```

Comment:
So the problem is the `scale-ci-base` uses `quay.io/pbench/pbench-agent-base-centos-7`, so we need to use `all` because to get the tools and the workloads.

Fixed the spelling error.

Reply:
Actually, I just pushed a commit which uses the `tools` layer so that we don't have to use `all`.  Let's see how that works.